src/sentry/snuba/discover.py

```diff
@@ -479,17 +431,14 @@ def query(
     """
     if not selected_columns:
         raise InvalidSearchQuery("No columns selected")
+    else:
+        # We clobber this value throughout this code, so copy the value
+        selected_columns = [c for c in selected_columns]
```

Comment:
```suggestion
        selected_columns = selected_columns[:]
```

I think this is functionally the same.